### PR TITLE
Localized content/bn/docs/tasks/tools/included/kubectl-whats-next.md

### PR DESCRIPTION
**Pull request by AdibReza**
Signed-off-by: Adib Reza <adib.orpo@gmail.com>
Localizes ``content/bn/docs/tasks/tools/included/kubectl-whats-next.md>``  to Bengali
This is an initial PR as it may need some improvements!

### CODE REVIEW

File: content/bn/docs/tasks/tools/included/kubectl-whats-next.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+description: "kubectl ইন্সটল করার পর কি হবে."
+headless: true
+_build:
+  list: never
+  render: never
+  publishResources: false
+---
+
+* [ইনস্টল করুন Minikube](https://minikube.sigs.k8s.io/docs/start/)
+* ক্লাস্টার তৈরি সম্পর্কে আরও জানতে দেখুন. [গাইড শুরু করা](/docs/setup/)
```
Comment (mitul3737) [inline content/bn/docs/tasks/tools/included/kubectl-whats-next.md:12]:
```suggestion
* ক্লাস্টার তৈরি সম্পর্কে আরও জানতে দেখুন [গাইড শুরু করা](/bn/docs/setup/) ফাইলটি ।
```

File: content/bn/docs/tasks/tools/included/kubectl-whats-next.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+* [আপনার অ্যাপ্লিকেশানটি কীভাবে লঞ্চ করবেন এবং প্রকাশ করবেন তা জানুন.](/docs/tasks/access-application-cluster/service-access-application-cluster/)
```
Comment (mitul3737) [inline content/bn/docs/tasks/tools/included/kubectl-whats-next.md:13]:
```suggestion
* [আপনার অ্যাপ্লিকেশানটি কীভাবে লঞ্চ করবেন এবং প্রকাশ করবেন তা জানুন ।](/bn/docs/tasks/access-application-cluster/service-access-application-cluster/)
```

File: content/bn/docs/tasks/tools/included/kubectl-whats-next.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+* আপনার যদি এমন একটি ক্লাস্টারে অ্যাক্সেসের প্রয়োজন হয় যা আপনি তৈরি করেননি, দেখুন
+  [Sharing Cluster Access document](/docs/tasks/access-application-cluster/configure-access-multiple-clusters/).
```
Comment (mitul3737) [inline content/bn/docs/tasks/tools/included/kubectl-whats-next.md:15]:
```suggestion
  [ক্লাস্টার অ্যাক্সেস নথি ভাগ করেন](/bn/docs/tasks/access-application-cluster/configure-access-multiple-clusters/).
```

File: content/bn/docs/tasks/tools/included/kubectl-whats-next.md [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+* এটা পড়ুন [kubectl রেফারেন্স ডক্স](/docs/reference/kubectl/kubectl/)
```
Comment (mitul3737) [inline content/bn/docs/tasks/tools/included/kubectl-whats-next.md:16]:
```suggestion
* [kubectl রেফারেন্স ডক্স](/bn/docs/reference/kubectl/kubectl/) পড়ুন ।
```

**Review by mitul3737 — CHANGES_REQUESTED** [review]

**Comment by mitul3737** [comment]
Tracking issue : #31677

File: content/bn/docs/tasks/tools/included/kubectl-whats-next.md (hunk 2 above)
Comment (sftim) [reply content/bn/docs/tasks/tools/included/kubectl-whats-next.md:13]: Does that target page exist in the [dev-1.23-bn.1](https://github.com/kubernetes/website/tree/dev-1.23-bn.1) branch?

File: content/bn/docs/tasks/tools/included/kubectl-whats-next.md (hunk 3 above)
Comment (sftim) [reply content/bn/docs/tasks/tools/included/kubectl-whats-next.md:15]: Does that target page exist in the [dev-1.23-bn.1](https://github.com/kubernetes/website/tree/dev-1.23-bn.1) branch?

File: content/bn/docs/tasks/tools/included/kubectl-whats-next.md (hunk 4 above)
Comment (sftim) [reply content/bn/docs/tasks/tools/included/kubectl-whats-next.md:16]: Does that target page exist in the [dev-1.23-bn.1](https://github.com/kubernetes/website/tree/dev-1.23-bn.1) branch?

**Comment by sftim** [comment]
/retitle Localize fragment of kubectl install page to Bengali

File: content/bn/docs/tasks/tools/included/kubectl-whats-next.md (hunk 2 above)
Comment (mitul3737) [reply content/bn/docs/tasks/tools/included/kubectl-whats-next.md:13]: Not yet @sftim  , thus I have removed the `bn` folder

File: content/bn/docs/tasks/tools/included/kubectl-whats-next.md (hunk 3 above)
Comment (mitul3737) [reply content/bn/docs/tasks/tools/included/kubectl-whats-next.md:15]: Not yet @sftim , thus I have removed the bn folder

File: content/bn/docs/tasks/tools/included/kubectl-whats-next.md (hunk 4 above)
Comment (mitul3737) [reply content/bn/docs/tasks/tools/included/kubectl-whats-next.md:16]: Not yet @sftim , thus I have removed the bn folder

**Review by mitul3737 — CHANGES_REQUESTED** [review]

**Comment by mitul3737** [comment]
/retitle Localized content/bn/docs/tasks/tools/included/kubectl-whats-next.md

File: content/bn/docs/tasks/tools/included/kubectl-whats-next.md [hunk 5]
```diff
@@ -0,0 +1,16 @@
+---
+title: "এরপর কি?"
+description: "kubectl ইন্সটল করার পর কি হবে."
```
Comment (mitul3737) [inline content/bn/docs/tasks/tools/included/kubectl-whats-next.md:3]:
```suggestion
description: "kubectl ইন্সটল করার পর যা হবে"
```

**Review by mitul3737 — CHANGES_REQUESTED** [review]

**Comment by mitul3737** [comment]
@AdibReza  Kindly complete this pr.

**Review by mitul3737 — APPROVED** [review]
/lgtm

**Pull request merged**